src/pretix/control/forms/orders.py

```diff
@@ -178,3 +179,31 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         locale_names = dict(settings.LANGUAGES)
         self.fields['locale'].choices = [(a, locale_names[a]) for a in self.instance.event.settings.locales]
+
+
+class OrderMailForm(forms.Form):
+    sendto = forms.CharField(label=_("Recipient"))
+    subject = forms.CharField(label=_("Subject"))
+    message = forms.CharField(label=_("Message"))
+
+    def __init__(self, *args, **kwargs):
+        event = kwargs.pop('event')
+        order = kwargs.pop('order')
+        super().__init__(*args, **kwargs)
+        self.fields['subject'] = I18nFormField(
+            widget=I18nTextInput, required=True,
+            locales=event.settings.get('locales')
+        )
+        self.fields['message'] = I18nFormField(
+            widget=I18nTextarea, required=True,
+            locales=event.settings.get('locales'),
+            help_text=_("Available placeholders: {due_date}, {event}, {order}, {order_date}, {order_url}, "
+                        "{invoice_name}, {invoice_company}"),
+            validators=[PlaceholderValidator(['{due_date}', '{event}', '{order}', '{order_date}', '{order_url}',
+                                              '{invoice_name}', '{invoice_company}'])]
+        )
+        self.fields['sendto'] = I18nFormField(
+            widget=I18nTextInput, required=True,
+            initial=order.email, locales=event.settings.get('locales')
+        )
+        self.fields['sendto'].widget.attrs['readonly'] = 'readonly'
```

Comment:
We don't really need I18nFormFields here, since when we write to one attendee only, we will only use one language. You can just use plain CharFields/TextFields/EmailFields and then you can define them like any other django form field